### PR TITLE
Update to rancher image, remove extra end in helper

### PR DESCRIPTION
**Pull request by brendarearden**
**Problem**
Kiali needs to be patched to have rancher mirrored images

Error occurring because too many end statements

**Solution**
Add rancher kiali image to kiali chart patch

Remove extra end statement in full name override helper for kiali chart patch 

**Issue**
https://github.com/rancher/rancher/issues/27943

### CODE REVIEW

**Review by paynejacob — APPROVED** [review]

**Review by cbron — CHANGES_REQUESTED** [review]
lgtm, but new request, can you add the hidden annotation to this chart.

**Comment by brendarearden** [comment]
@cbron I have added the hidden annotation

**Review by cbron — APPROVED** [review]
lgtm, thanks

File: packages/rancher-kiali-server/rancher-kiali-server.patch [hunk 1]
```diff
@@ -37,22 +37,24 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-kiali-server/charts-original/C
...
    {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
  {{- else }}
    {{- $name := default .Chart.Name .Values.nameOverride }}
 -  {{- if contains $name .Release.Name }}
 -    {{- .Release.Name | trunc 63 | trimSuffix "-" }}
 -  {{- else }}
 -    {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
-+    {{- printf "%s" $name | trunc 63 | trimSuffix "-" }}
-   {{- end }}
+-  {{- end }}
```
Comment (rmweir) [inline packages/rancher-kiali-server/rancher-kiali-server.patch:52]: The word "end" is still here or am I reading this wrong?

Comment (brendarearden) [reply packages/rancher-kiali-server/rancher-kiali-server.patch:52]: Its `+` a `-  {{- end }}`, so the patch would remove the end

**Review by rmweir — APPROVED** [review]
LGTM

**Pull request merged**